examples/seek_viewer.cpp

```diff
@@ -22,11 +22,13 @@ void handle_sig(int sig) {
 }
 
 // Function to process a raw (corrected) seek frame
-void process_frame(Mat &inframe, Mat &outframe, float scale, int colormap, int rotate) {
+void process_frame(Mat &inframe, Mat &outframe, float scale, int colormap, int rotate, int cont, int bright) {
     Mat frame_g8, frame_g16; // Transient Mat containers for processing
-
+   if(cont == 0){
```

Comment:
Incorrect spacing, all the changes need to go through formatting to bring them in line with the established style